File: app/src/main/java/com/deepquotes/Quotes.java
package com.deepquotes;

import android.app.job.JobParameters;

public class Quotes {


    public static final int UPDATE_TEXT = 123456;
    private String text;
    private String author;
    private JobParameters jobParameters;

    public Quotes(String text, String author,JobParameters jobParameters) {
        this.text = text;
        this.author = author;
        this.jobParameters = jobParameters;
    }

    public String getAuthor() {
        return author;
    }

    public Quotes(String text, String author) {
        this.text = text;
        this.author = author;
    }

    public String getText() {
        return text;
    }

    public JobParameters getJobParameters() {
        return jobParameters;
    }

}

File: app/src/main/java/com/deepquotes/Quotes.kt
package com.deepquotes

import android.app.job.JobParameters

class Quotes {
    var text: String
        private set
    var author: String
        private set
    var jobParameters: JobParameters? = null
        private set

    constructor(text: String, author: String, jobParameters: JobParameters?) {
        this.text = text
        this.author = author
        this.jobParameters = jobParameters
    }

    constructor(text: String, author: String) {
        this.text = text
        this.author = author
    }

    companion object {
        const val UPDATE_TEXT = 123456
    }
}